Branch-SDK/Branch-SDK/BNCPreferenceHelper.m

```diff
@@ -181,15 +181,7 @@ + (NSInteger)getRetryCount {
 }
 
 + (NSString *)getAppKey {
-    NSString *ret = [[[NSBundle mainBundle] infoDictionary] objectForKey:KEY_APP_KEY];
-    if (!ret || ret.length == 0) {
-        // for backward compatibility
-        ret = [BNCPreferenceHelper readStringFromDefaults:KEY_APP_KEY];
```

Comment:
I was leaning toward it not working for either, so that this problem would be rooted out potentially before going live -- either the tester would realize because it'd blow up for them, or the beta testers would see it. Also, when was this feature present in Branch, any idea? It seems we moved away from saving into defaults a while ago. Will try to find reference.